File: C/Chapter11/Task5.c
//
// Create a program which breaks down the given number (integer) to bills and coins.
//
#include <stdio.h>

int main()
{
    int amount;
    int division;
    printf("Enter the value\n");
    scanf(" %i", &amount);
    division = amount / 200;
    printf("%i x 200\n", division);
    amount -= division*200;
    division = amount / 100;
    printf("%i x 100\n", division);
    amount -= division*100;
    division = amount / 50;
    printf("%i x 50\n", division);
    amount -= division*50;
    division = amount / 20;
    printf("%i x 20\n", division);
    amount -= division*20;
    division = amount / 10;
    printf("%i x 10\n", division);
    amount -= division*10;
    division = amount / 5;
    printf("%i x 5\n", division);
    amount -= division*5;
    division = amount / 2;
    printf("%i x 2\n", division);
    amount -= division*2;
    division = amount / 1;
    printf("%i x 1\n", division);
    amount -= division*1;
}

File: Java/Chapter03/src/me/michal/assignments/lab03/Task5.java
//
// Create a program which prints every value of the power of 3 smaller than the entered number.
//
package me.michal.assignments.lab03;

import java.util.Scanner;

import static java.lang.Math.pow;

public class Task5 {
    public static void main(String[] args) {
        int number;
        Scanner input = new Scanner(System.in);
        System.out.println("Enter the non-negative integer");
        number = input.nextInt();
        if (number <= 0)
            System.err.println("Vrong value!");
        else {
            for (int i = 1; pow(3, i) < number; i++) {
                System.out.println(pow(3, i));
            }
        }
    }
}
